index.html

```diff
@@ -128,16 +158,18 @@ <h3>Project name goes here</h3>
                     </button>
                 </a>
             </div>
-        </div>
-       </div>
-    </section>
+          </div>
+       
+        </section>
     <section class="about">
+        <div class="abouthead">
         <h2>About</h2>
         <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
         <button type="button">
             Get my CV
             <img src="images/ic_download.svg">
         </button>
+    </div>
```

Comment:
- [ ] Kindly try to fix this section, the elements are overlapping each other, the cards on the left are not aligned correctly, the button is on top of the text. The typography styling doesn't match either, kindly increase the `font-size` of the header "About" and decrease the `font-size` of the paragraph below it 👍 
- [ ] Kindly add the interaction styles of the button (hover, click...) 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/200124891-7038327b-ab34-4a5f-a597-06d15b0cd452.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/200124898-70bedce9-051f-4ec8-ba26-f6b40dcd9ed7.png)

Reply:
Fixed.